en/docs/get-started/about-this-release.md

```diff
@@ -61,6 +61,12 @@ It is now available to download from [here](https://wso2.com/api-management/#).
 - Defaulting code and configs to do with event based
 -->
 
+### Deprecated features and functionalities
```

Comment:
This should be "Removed Features" right? Is it still shipped with 4.0.0?

Reply:
Yes, this will be still shipped with 4.0.0, but deprecated.